store/event-store/src/test/java/org/calrissian/accumulorecipes/eventstore/impl/AccumuloEventStoreTest.java

```diff
@@ -136,6 +136,8 @@ public void testVisibility() {
         assertEquals(1, Iterables.get(actualEvent1, 1).size());
     }
 
+
+    @Ignore // ignoring for now. We may not need tuples to expire
```

Comment:
Yeah, we did talk about it, but there isn't really any record in the project of why it was removed.  I also didn't remember it would be part of this pull request.  My only suggestion is that if it will be removed it should be described in a ticket and linked to this pull request so others (or future me) knows why it was removed and where.